tracee-ebpf/tracee/consts.go

```diff
@@ -938,7 +944,7 @@ var EventsIDToParams = map[int32][]external.ArgMeta{
 	SysEnterEventID:               {{Type: "int", Name: "syscall"}},
 	SysExitEventID:                {{Type: "int", Name: "syscall"}},
 	SchedProcessForkEventID:       {{Type: "int", Name: "parent_tid"}, {Type: "int", Name: "parent_ns_tid"}, {Type: "int", Name: "child_tid"}, {Type: "int", Name: "child_ns_tid"}},
-	SchedProcessExecEventID:       {{Type: "const char*", Name: "cmdpath"}, {Type: "const char*", Name: "pathname"}, {Type: "const char**", Name: "argv"}, {Type: "const char**", Name: "env"}, {Type: "dev_t", Name: "dev"}, {Type: "unsigned long", Name: "inode"}, {Type: "int", Name: "invoked_from_kernel"}, {Type: "unsigned long", Name: "ctime"}},
+	SchedProcessExecEventID:       {{Type: "const char*", Name: "cmdpath"}, {Type: "const char*", Name: "pathname"}, {Type: "const char**", Name: "argv"}, {Type: "const char**", Name: "env"}, {Type: "dev_t", Name: "dev"}, {Type: "unsigned long", Name: "inode"}, {Type: "int", Name: "invoked_from_kernel"}, {Type: "unsigned long", Name: "ctime"}, {Type: "unsigned int", Name: "stdin"}},
```

Comment:
`{Type: "unsigned int", Name: "stdin"}` doesn't mean anything to anyone. can we make it an enum? the work in https://github.com/aquasecurity/libbpfgo/pull/89 might be related

Reply:
thanks @itaysk .
i agree, it doesn't mean a lot. i did add an enum of the supported file types in consts.go, and i've added the ParseStdinType() func in argprinters.go.
i can change the name of the argument to something like "stdin_file_type", do you think that would help?
regarding the type "unsigned int" - i can't change that because this indicated what is being passed to userSpace from the bpf code.

should the argprinters func, ParseStdinType(), be part of libbpfgo? 
it seems to me like it doesn't belong there because we don't support all file types in tracee currently - so it's not a generic kernel thing yet.
wdyt?

Reply:
> i did add an enum of the supported file types in consts.go

You added some int consts, not what I meant. Go doesn't have a proper enums so I linked that libbpfgo PR as example of "enums" in go (basically, type + stringer)

> i can't change that because this indicated what is being passed to userSpace from the bpf code

Actually that's not true anymore. it's a hint to help deserialize the arg value as accurately as possible, see https://github.com/aquasecurity/tracee/issues/1029 for another example.

> should the argprinters func, ParseStdinType(), be part of libbpfgo?

Only if this is something that some syscall accept/returns